src/libexec/zmmsgtrace

```diff
@@ -461,6 +461,7 @@ sub doit {
                         $ref = $obj->{recipList}->{$recip} = {};
                     }
 
+                    $ref->{messageId}  ||= "[unknown:$key]";
```

Comment:
```suggestion
                    $ref->{messageId}  ||= "[unknown:$key]";
                    $obj->{messageId}  ||= $ref->{messageId};
```
We need to also initialize the message ID in $obj to avoid the warnings.